examples/sveltekit-ts/package.json

```diff
@@ -25,19 +25,19 @@
 	},
 	"devDependencies": {
 		"@playwright/test": "^1.37.1",
-		"@sveltejs/adapter-static": "^2.0.3",
-		"@sveltejs/adapter-node": "^1.3.1",
-		"@sveltejs/kit": "^1.24.1",
-		"@types/cookie": "^0.5.2",
-		"@typescript-eslint/eslint-plugin": "^6.13.2",
-		"@typescript-eslint/parser": "^6.13.2",
+		"@sveltejs/adapter-static": "^3.0.0",
+		"@sveltejs/adapter-node": "^2.0.0",
+		"@sveltejs/kit": "^2.0.0",
+		"@types/cookie": "^0.6.0",
+		"@typescript-eslint/eslint-plugin": "^6.14.0",
+		"@typescript-eslint/parser": "^6.14.0",
 		"@vite-pwa/sveltekit": "workspace:*",
-		"eslint": "^8.49.0",
-		"eslint-plugin-svelte": "^2.34.0",
-		"svelte": "^4.2.0",
-		"svelte-check": "^3.5.1",
+		"eslint": "^8.55.0",
+		"eslint-plugin-svelte": "^2.35.1",
+		"svelte": "^5.0.0-next.25",
+		"svelte-check": "^3.6.2",
 		"tslib": "^2.6.2",
-		"typescript": "^5.2.2",
+		"typescript": "^5.3.3",
 		"vitest": "^0.34.4"
```

Comment:
this can be 1.0.0 now

Reply:
yeah, I need to check it, I forgot to update Vitest...